eth2deposit/credentials.py

```diff
@@ -57,6 +60,8 @@ def withdrawal_credentials(self) -> bytes:
 
     @property
     def deposit_message(self) -> DepositMessage:
+        if self.amount > MAX_DEPOSIT_AMOUNT:
+            raise ValidationError(f"{self.amount / ETH2GWEI} ETH is more than the maximum allowed deposit.")
```

Comment:
nitpicking: "maximum allowed deposit" here is for the deposit-cli, not for the deposit contract.

maybe:

```suggestion
            raise ValidationError(f"{self.amount / ETH2GWEI} ETH is more than the maximum allowed deposit in deposit-cli.")
```

or
```suggestion
            raise ValidationError(f"{self.amount / ETH2GWEI} ETH is more than the minimum required deposit.")
```

Reply:
You are right, this is more accurate.

I've updated the exception to be raised if the amount is too high, or too low and it now has a more accurate error message.